src/xcode/ENA/ENA/Source/View Helpers/AccessibilityIdentifiers.swift

```diff
@@ -539,6 +539,10 @@ enum AccessibilityIdentifiers {
 			static let title = "AppStrings.Statistics.Card.IntensiveCare.title"
 			static let infoButton = "AppStrings.Statistics.Card.IntensiveCare.infoButton"
 		}
+		enum Combined7DayIncidence {
+			static let title = "AppStrings.Statistics.Card.Combined7DayIncidence.title"
+			static let infoButton = "AppStrings.Statistics.Card.Combined7DayIncidence.infoButton"
+		}
```

Comment:
Old two cards can be deleted here as well, and the illustrations from the asset catalogue

Reply:
done